scripts/_check_docs_utils.py

```diff
@@ -0,0 +1,190 @@
+# coding: utf-8
```

Comment:
To double-check: all code in this file was originally written by us?

Reply:
Yes. The initial helper functions are similar but modified. I have changed the docstrings as well.

Reply:
I'm confused what you mean by "similar but modified", can you elaborate?

Reply:
The pylint functions check for all Docstrings style. This has been modified to check only for our modified Google Docstrings style.